shared/src/test/diff/mlscript/Methods2.mls

```diff
@@ -68,22 +67,22 @@ l.Size
 l.Map (fun x -> mul x 2)
 //│ res: List['B .. 'B | int]
 
-o = l.HeadOption
```

Comment:
Small detail: let's not swap these needlessly.

Reply:
The pattern matching below only fails with an `Option`. If `Cons.HeadOption` is called explicitly, the result would be a `Some` and the match would succeed. So I swapped them to preserve the error below.

Reply:
Oh yes good point, I had missed that 👍